openhtf/util/__init__.py

```diff
@@ -56,8 +57,24 @@ def TimeMillis():  # pylint: disable=invalid-name
   return int(time.time() * 1000)
 
 
-def convert_to_dict(obj, ignore_keys=tuple()):
-  """Recursively convert namedtuples to dicts."""
+def ConvertToBaseTypes(obj, ignore_keys=tuple()):
+  """Recursively convert objects into base types, mostly dicts and strings.
```

Comment:
...dicts and primitives

Otherwise, one might think this stringifies enums, ints and floats, which it doesn't